### PR TITLE
feat: update readme and fix some issues

### PR DESCRIPTION
**Pull request by CoderMungan**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -60,6 +60,7 @@ docker-compose up
 - All applications are inside the `Apps` folder.
 - You can access application files in the `templates` folder accordingly.
 - The `static` folder was completely collectstatic.
+- Add the `tavsanli-41bb0a89bf6a.json` file from the discord.
```
Comment (mciray) [inline README.md:63]: gj

File: archaeologyMain/apps/buluntu/models.py [hunk 2]
```diff
@@ -1,15 +1,17 @@
...
 from django.db import models
-
 from dotenv import load_dotenv
 
 load_dotenv()
 
-from gdstorage.storage import GoogleDriveStorage, GoogleDrivePermissionType, GoogleDrivePermissionRole, GoogleDriveFilePermission
+from gdstorage.storage import (GoogleDriveFilePermission,
+                               GoogleDrivePermissionRole,
+                               GoogleDrivePermissionType, GoogleDriveStorage)
```
Comment (mciray) [inline archaeologyMain/apps/buluntu/models.py:14]: gj

File: archaeologyMain/static/js/tinyhtml.js [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+  plugins: "spellchecker,directionality,paste,searchreplace",
+  language: "tr",
+  branding: false,
+  directionality: "{{ directionality }}",
+  spellchecker_languages : "{{ spellchecker_languages }}",
+  spellchecker_rpc_url : "{{ spellchecker_rpc_url }}",
+  skin: (window.matchMedia('(prefers-color-scheme: dark)').matches ? 'oxide-dark' : 'oxide'),
+  content_css: (window.matchMedia('(prefers-color-scheme: dark)').matches ? 'dark' : 'default'),
+  menubar: false
+});
```
Comment (mciray) [inline archaeologyMain/static/js/tinyhtml.js:13]: well done

**Review by mciray — APPROVED** [review]
LGTM

**Review by AysKrimn — APPROVED** [review]

**Pull request merged**